spec/controllers/api/v1/admin/account_actions_controller_spec.rb

```diff
@@ -1,55 +0,0 @@
-# frozen_string_literal: true
-
-require 'rails_helper'
-
-RSpec.describe Api::V1::Admin::AccountActionsController do
-  render_views
-
-  let(:role)   { UserRole.find_by(name: 'Moderator') }
-  let(:user)   { Fabricate(:user, role: role) }
-  let(:scopes) { 'admin:read admin:write' }
-  let(:token)  { Fabricate(:accessible_access_token, resource_owner_id: user.id, scopes: scopes) }
-  let(:account) { Fabricate(:account) }
-
-  before do
-    allow(controller).to receive(:doorkeeper_token) { token }
-  end
-
-  describe 'POST #create' do
-    context 'with type of disable' do
-      before do
-        post :create, params: { account_id: account.id, type: 'disable' }
-      end
-
-      it_behaves_like 'forbidden for wrong scope', 'write:statuses'
-      it_behaves_like 'forbidden for wrong role', ''
-
-      it 'returns http success' do
-        expect(response).to have_http_status(200)
-      end
-
-      it 'performs action against account' do
-        expect(account.reload.user_disabled?).to be true
-      end
-
-      it 'logs action' do
-        log_item = Admin::ActionLog.last
-
-        expect(log_item).to_not be_nil
-        expect(log_item.action).to eq :disable
-        expect(log_item.account_id).to eq user.account_id
-        expect(log_item.target_id).to eq account.user.id
-      end
```

Comment:
This part of the spec is not reproduced in the request spec. Why?

Reply:
My bad, I thought those belonged to the model specs. But I think it makes sense to include them in the request specs as well since they serve as integration tests. Just added them in [2667527](https://github.com/mastodon/mastodon/pull/25514/commits/26675274f4fff37b0774779c091575ee9976feef)

Reply:
It's currently not handled by the model, but by the controllers!